pkg/addons/gcpauth.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The Kubernetes Authors All rights reserved.
+Copyright 2021 The Kubernetes Authors All rights reserved.
```

Comment:
same here file name `addons_gcp_auth.go`

Reply:
we're already in the addons package and directory, why does the file name need to be longer?